Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -721,8 +721,8 @@ private OrderResponse PreOrderChecksImpl(SubmitOrderRequest request)
                 return OrderResponse.Error(request, OrderResponseErrorCode.SecurityHasNoData, "There is no data for this symbol yet, please check the security.HasData flag to ensure there is at least one data point.");
             }
 
-            //We've already processed too many orders: max 100 per day or the memory usage explodes
-            if (Transactions.OrdersCount > _maxOrders)
+            // We've already processed too many orders: max 10k
+            if (!LiveMode && request.OrderId > _maxOrders)
```

Comment:
great catch with the live mode check here -- I did a quick check and didn't find us properly setting this to something large like `int.MaxValue` in live mode... so _technically_ it looks like a live algo would have stopped placing trades once it hit this mark

Reply:
Thanks!